client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -590,7 +590,7 @@ public final void extractRPCSchema(){
      * parse seeded tests for RPC
      * @return a list of tests, and each test is a list of RCPActionDto
```

Comment:
update docs... now we do not return a list, but a map (also clarify what are the string keys here)